lib/command/Command.js

```diff
@@ -354,15 +384,27 @@ class Command {
         }
     }
 
-    executeCommand(msg, args) {
+    async executeCommand(msg, args) {
         if(this.hooks.postCheck) {
-            this.hooks.postCheck(msg, args, true);
+            if (this.hooks.postCheck(msg, args, true) !== undefined) {
+                const response = await this.hooks.postCheck(msg, args, true);
+                msg = response.msg ? response.msg : msg;
+                args = response.args ? response.args : args;    
+            } else {
+                this.hooks.postCheck(msg, args, true);
+            }
         }
 
         let ret = this.execute(msg, args);
 
         if(this.hooks.postExecution) {
-            this.hooks.postExecution(msg, args, true);
+            if (this.hooks.postExecution(msg, args, true) !== undefined) {
```

Comment:
Do we really need to implement this for `postExecution` since `msg` and `args` aren't used afterwards?